<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameRuleWidget</class>
 <widget class="QWidget" name="NameRuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NameRuleWidget</string>
  </property>
  <property name="_vanillaStyle_Patch" stdset="0">
   <string notr="true">CleanButtonPatch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="NameRuleEditWidget" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="labelPreview">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
            <iconset theme=":/icon/reset.svg"/>
        </property>
        <property name="_vanillaStyle_Patch" stdset="0">
         <string notr="true">CleanButtonPatch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="NameRuleSelectWidget" name="listNameRule">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NameRuleSelectWidget</class>
   <extends>QListWidget</extends>
   <header>SUI/NameRule/NameRuleSelectWidget.h</header>
  </customwidget>
  <customwidget>
   <class>NameRuleEditWidget</class>
   <extends>QLineEdit</extends>
   <header>SUI/NameRule/NameRuleEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
